site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -129,10 +129,10 @@
 </script>
 
 {% macro bbcode_buttons() %}
```

Comment:
Maybe it is no longer called `bbcode_buttons()`

Reply:
I'll take a look!

Reply:
I mean it's no longer bbcode.